file-store/src/test/scala/com.expedia.blobs.stores.io/FileStoreSpec.scala

```diff
@@ -1,17 +1,23 @@
-package com.expedia.blobs.core.io
+package com.expedia.blobs.stores.io
```

Comment:
Can you check the folder name? It is a folder named `com.expedia.blobs.stores.io` and not `com/expedia/blobs/stores/io`

Reply:
It's `com/expedia/blobs/stores/io`

Reply:
![2019-06-04_05-45-01](https://user-images.githubusercontent.com/148128/58880037-d3976000-868b-11e9-9b8b-4eee983d7eed.png)